MinVerTests/Versioning.cs

```diff
@@ -139,21 +138,5 @@ public static void NoRepo(string path, Version version)
             "Then the version is 0.0.0-alpha.0"
                 .x(() => Assert.Equal("0.0.0-alpha.0", version.ToString()));
         }
-
-        [Scenario]
-        public static void WorkingDir(string path, Version version)

```

Comment:
This scenario was named incorrectly. It's actually for the _repo_ (`.git`) and all the _other_ scenarios are for the working dir.